### PR TITLE
ci: update calls to slack notify to include secrets

### PR DESCRIPTION
**Pull request by gbockus-sf**
### What does this PR do?
Follow up after testing to  #4546. Updates slack notify action to take secrets for posting to slack. 

### What issues does this PR fix or reference?
@W-11824084@

### Functionality Before
silent failure to post to slack

### Functionality After
successfully posts to slack

### CODE REVIEW

File: .github/workflows/createReleaseBranch.yml [hunk 1]
```diff
@@ -67,7 +67,10 @@ jobs:
   slack_notification:
     if: ${{ always() }}
     needs: create_branch
-    uses: ./.github/workflows/slack-notification.yml
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: 
+      SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
+      IDEE_RELEASE_ALERT_SLACK_WEBHOOK: ${{ secrets.IDEE_RELEASE_ALERT_SLACK_WEBHOOK }}
```
Comment (gbockus-sf) [inline .github/workflows/createReleaseBranch.yml:73]: this was the trick.  It needs to have the secrets passed.  Probably could have also used [inherit](https://github.blog/changelog/2022-05-03-github-actions-simplify-using-secrets-with-reusable-workflows/)

File: .github/workflows/testSlackWorkflow.yml [hunk 2]
```diff
@@ -8,10 +8,11 @@ jobs:
...
     secrets: 
       SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
+      IDEE_RELEASE_ALERT_SLACK_WEBHOOK: ${{ secrets.IDEE_RELEASE_ALERT_SLACK_WEBHOOK }}
     with:
       title: "Test: Release Branch v56.7.1 (patch) successfully created"
       failedEvent: "Create Release Branch"
       successfulEvent: "https://github.com/forcedotcom/salesforcedx-vscode/tree/release/v56.7.1"
       type: "created"
-      result: "Success"
+      result: "failure"
```
Comment (gbockus-sf) [inline .github/workflows/testSlackWorkflow.yml:17]: updated the test action to send a failure.

File: .github/workflows/createReleaseBranch.yml [hunk 3]
```diff
@@ -67,7 +67,10 @@ jobs:
   slack_notification:
     if: ${{ always() }}
     needs: create_branch
-    uses: ./.github/workflows/slack-notification.yml
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: 
+      SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
```
Comment (CristiCanizales) [inline .github/workflows/createReleaseBranch.yml:72]: maybe have the same naming convention with the 2 secrets?

File: .github/workflows/publishBetaRelease.yml [hunk 4]
```diff
@@ -81,7 +81,10 @@ jobs:
   slack_notification:
     if: ${{ always() }}
     needs: create_git_tag
-    uses: ./.github/workflows/slack-notification.yml
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: 
+      SLACK_WEBHOOK_URL: ${{ secrets.VSCODE_SLACK_WEBHOOK }}
```
Comment (CristiCanizales) [inline .github/workflows/publishBetaRelease.yml:86]: same here with the naming convention

File: .github/workflows/testSlackWorkflow.yml (hunk 2 above)
Comment (CristiCanizales) [reply .github/workflows/testSlackWorkflow.yml:17]: We need to get rid of this before merging right?

**Review by CristiCanizales — COMMENTED** [review]
Awesome! Thank you 🙌🏼 Just a few comments. Also thank you for renaming the files, I rather cammelCase than kebab-case (?

File: .github/workflows/createReleaseBranch.yml (hunk 1 above)
Comment (gbockus-sf) [reply .github/workflows/createReleaseBranch.yml:73]: yeah me to...will update

File: .github/workflows/testSlackWorkflow.yml (hunk 2 above)
Comment (gbockus-sf) [reply .github/workflows/testSlackWorkflow.yml:17]: nah it doesn't hurt anything to be in the repo briefly.  I'll delete it after we verify the flow 👍

**Review by CristiCanizales — APPROVED** [review]
Looks good to me! 🙌🏼

**Pull request merged**